.github/workflows/wiki.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Wiki-related'
```

Comment:
This is the name we see in the Github Actions. So it should be a bit nicer, like 'Wiki Update'.